src/components/DataSection/DataSection.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import './DataSection.scss';
+import { getLocation } from '../../commons/helpers';
+import Card from '../Card/Card';
+import { Tabs } from '../../statics/Statics';
+
+function DataSection ({activeTab}) {
+    const [data, setData] = useState([]);
+    const [loading, setLoading] = useState(false);
+
+    useEffect(() => getData(), [activeTab])
+
+    const callback = (results, status) => {
+     if (status === window.google.maps.places.PlacesServiceStatus.OK) {
+            setData(results)
+            setLoading(false)
+        }
+    }
+    const getData = () => {
+        setLoading(true)
+        getLocation().then(({ coords: {latitude, longitude} }) => {
```

Comment:
You're fetching the location twice, once in the header and then here. Isn't it better to cache it?
Also, what if the user has not allowed the location access? Should you fall back to a default/generic location?

Reply:
I think it's better to not fallback to a default location as it gives a false impression of having incorrect nearby places.

Reply:
Agreed with the caching point. Will make changes.